src/main/java/org/mockito/internal/stubbing/defaultanswers/ReturnsMoreEmptyValues.java

```diff
@@ -61,13 +36,10 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
         Class<?> returnType = invocation.getMethod().getReturnType();
         return returnValueFor(returnType);
     }
-    
+
     Object returnValueFor(Class<?> type) {
         if (type == String.class) {
             return "";
```

Comment:
As general feedback, not particularly to this PR, I'm +1 to returning empty String. We can poll more feedback from the community. Empty String seems like a better default, avoiding potential NPE. The idea behind the defaults is to return the most sensible value if the user does not care about the mock.